main.py

```diff
@@ -11,5 +14,23 @@ def view_store_route():
     return {"store_data": results}
 
 
+@app.get("/korpa")
+def view_cart_route():
+    results = view_cart()
+    return {"cart_data": results}
+
+
+@app.post("/add_cart")
+def add_cart_route(kupac: Buyer, product: Product):
```

Comment:
Kupac should be stored in some place. For example, create logged_in.csv where you will store the buyer who is logged in

Reply:
i did it, i will send leter.